### ISSUE TITLE
expose generic matcher method for request

### ISSUE THREAD

**Issue by musketyr** [cjstehno/ersatz#87]
there should be method accepting `Matcher<ClientRequest>` on `Request` for more complex/not yet supported client request matching.

**Comment by cjstehno** [comment]
Makes sense. Something like:

```
Matcher<ClientRequest> crMatcher = ...

get('/foo').matches(crMatcher)
```

**Comment by cjstehno** [comment]
This has been fixed in development (1.6.2)
